templates/fine-tune-llm/training_configs/llama-2-13b-4k-4xg5_12xlarge.yaml

```diff
@@ -0,0 +1,16 @@
+model_id: meta-llama/Llama-2-13b-hf # <-- change this to the model you want to fine-tune
+train_path: s3://air-example-data/gsm8k/train.jsonl # <-- change this to the path to your training data
+valid_path: s3://air-example-data/gsm8k/test.jsonl # <-- change this to the path to your validation data. This is optional
+context_length: 4096 # <-- change this to the context length you want to use
+num_devices: 8
+num_epochs: 10 # <-- change this to the number of epochs that you want to train for
+train_batch_size_per_device: 8
+eval_batch_size_per_device: 8
+learning_rate: 5e-6
+num_checkpoints_to_keep: 1
+output_dir: /mnt/local_storage
+deepspeed:
+  config_path: deepspeed_configs/zero_3_llama_2_13b.json
+flash_attention_2: true
+worker_resources:
+  g5.12xlarge: 1
```

Comment:
do we not add num_gpus for worker_resources?

Reply:
I am actually not familiar with `num_gpus` and haven't needed it.

The job compute config specifies the total num of GPUs that each custom resource has

example: https://github.com/anyscale/templates/pull/59/files#diff-ab6f122aecb021b10df54557219614fb35498f290f859fddf192471ed883c379R37